Tests/RxCocoaTests/Observable+BindTests.swift

```diff
@@ -30,7 +30,32 @@ extension ObservableBindTest {
         XCTAssertEqual(events, [
             .next(1),
             .completed()
-            ])
+        ])
+    }
+
+    func testBindToObservers() {
+        var events1: [Recorded<Event<Int>>] = []
+        var events2: [Recorded<Event<Int>>] = []
+
+        let observer1: AnyObserver<Int> = AnyObserver { event in
+            events1.append(Recorded(time: 0, value: event))
+        }
+
+        let observer2: AnyObserver<Int> = AnyObserver { event in
+            events2.append(Recorded(time: 0, value: event))
+        }
+
+        _ = Observable.just(1).bind(to: observer1, observer2)
+
+        XCTAssertEqual(events1, [
+            .next(1),
+            .completed()
+        ])
+
+        XCTAssertEqual(events2, [
+            .next(1),
+            .completed()
+        ])
```

Comment:
It seems to me that we should remove all tests with plural, like `testBindToOptionalObserver`. Don't see a need for them.

Reply:
How so ? We need to always validate that doing `bind(to: relay)` and `bind(to: relay1, relay2)` works in our tests.